src/template.js

```diff
@@ -11848,6 +11852,10 @@ module.exports = {
                                 const: false
                             }
                         ]
+                    },
+                    'help-understanding-care': {
+                        description:
+                            '{% from "components/details/macro.njk" import govukDetails %}{{ govukDetails({summaryText: "Help with physical aids",html: \'<h2 class="govuk-heading-s">Care</h2><p class="govuk-body">This means they cannot do daily tasks themselves. For example, care for the victim may include but is not limited to.</p><ul class="govuk-list govuk-list--bullet"><li>preparing meals</li><li>helping them to eat</li><li>helping them go to the bathroom</li><li>taking medication</li><li>administering other medical treatments</li></ul><h2 class="govuk-heading-s">Supervision</h2><p class="govuk-body">This means they pose a substantial danger to themselves or others without supervision.</p>\'})}}'
```

Comment:
Is this the correct use of `aids`? To me, "Physical Aids" implies we are talking about apparatus that helps a person in their daily life. Whereas the subsequent paragraph and bullet points imply that we are referring to a person helping the applicant here - i.e. they would be an "aide" or "aides".

I love a bit of pedantry

Reply:
That's a good point and well spotted. It matches the prototype though and one for GB?

Reply:
my bad, this got messed up when I fixed the linting issue I had.